### DISCUSSION TITLE
How to backup?

### DISCUSSION THREAD

**Discussion by keblek** [FiloSottile/age#323] [Q&A]
What is the recommended way to back the private keys up (that age generates)?

**Comment by FiloSottile** [comment]
They are plaintext files, so you can back them up with whatever system you trust. If you want, you can passphrase encrypt them with `age -p`.

**Reply by keblek** [reply]
Thanks, I was curious as well if you have an opinion about using mnemonic phrases to back up these keys (in case someone wants to back up to paper).  Or do you think this is excessive given the use case?

**Comment by AnimMouse** [comment]
KeePass
